qa/rpc-tests/mempool_packages.py

```diff
@@ -151,6 +152,7 @@ def run_test(self):
         # Test reorg handling
         # First, the basics:
         self.nodes[0].generate(1)
+        print("syncing blocks")
```

Comment:
nit: is this new print useful?

Reply:
It is, because the potential nondeterministic test failure that I've attempted to fix in https://github.com/zcash/zcash/pull/6542/commits/b5d0bb93a654b697c14082de93d163e357298db1, happens just after this.